osu.Game/Screens/Play/Player.cs

```diff
@@ -438,7 +439,7 @@ protected override void UpdateBackgroundElements()
                 .FadeTo(storyboardVisible && BackgroundOpacity > 0 ? 1 : 0, BACKGROUND_FADE_DURATION, Easing.OutQuint);
 
             if (storyboardVisible && beatmap.Storyboard.ReplacesBackground)
-                Background?.FadeTo(0, BACKGROUND_FADE_DURATION, Easing.OutQuint);
+                Background?.FadeColour(OsuColour.Gray(0f), BACKGROUND_FADE_DURATION, Easing.OutQuint);
```

Comment:
Color4.Black

Reply:
would need to add another using osuTK.Graphics for this too since its not needed in this file yet

Reply:
correct